src/Hazelcast.Net/Protocol/CustomCodecs/HazelcastJsonValueCodec.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
```

Comment:
If we modify codecs that are generated by the `protocol` repository then the PR should also update the `protocol` submodule state to the commit that was used to produce the new codecs. Is this possible (or is this the confusing situation we discussed before and we have to accept it for now)?

Reply:
This is a good question. I don't think it's discussed. I believe commiting from submodule could be confusing but we may force the owner of the PR to include related `protocol` PR link in the client code PR. So, at least, we can confirm that flow is not violated or causing confusion during review. Using submodule read-only may prevent possible synchronization problems.

Reply:
I am lost. I think the `protocol` repo is now dirty. Let's merge this as-is and we'll clean it up later. In *theory* we should (1) have a `protocol` repo PR, (2) get it merged, (3) update our repo to point to that new commit, (4) create a PR in our repo once everything is stable.

Reply:
I have synced the protocols and opened a PR https://github.com/hazelcast/hazelcast-client-protocol/pull/421, after merging, will reflect the up to date protocol submodule here.

Reply:
Updated the protocol and regenerated all codecs with latest version of the protocol by `hz.ps1 generate-codes`. All generated codecs are fine. Also, checked the all year bumps on the disclaimer one by one.  Just realized `HazelcastJsonValue` has a missing property `Value`, and added it. I think it is ready to go @zpqrtbnk